src/components/shortcircuit-analysis-result.js

```diff
@@ -55,11 +131,11 @@ const ShortCircuitAnalysisResult = ({ result }) => {
                     limitMin:
                         lv.limitType === 'LOW_SHORT_CIRCUIT_CURRENT'
                             ? lv.limit
-                            : NaN,
+                            : null,
                     limitMax:
                         lv.limitType === 'HIGH_SHORT_CIRCUIT_CURRENT'
                             ? lv.limit
-                            : NaN,
+                            : null,
                     limitName: lv.limitName,
                     current: lv.value,
```

Comment:
why no linkedElementId here ? These violations will be separated from the fault result ?